dist/success-message/ds4/success-message.css

```diff
@@ -66,6 +24,24 @@
   text-align: center;
   width: 100%;
 }
+.success-message .btn--secondary,
```

Comment:
Is there any way we can reduce the number of btn overrides here? I know this is only a few, and not really that bad, but our goal is to see if both notice and success-message can use btn right out of the box using only its modifiers.

Reply:
Yes I agree, we should only use the button without any secondary.

Reply:
`border-color` - the default is blue, we need black
`color` - the default is blue, we need black

But I'll remove the `text-decoration`.  I didn't see that it was already in the default.

Reply:
I think if we use just `btn` without anything else. 
If I have a `<button class="btn"/>` This is what it looks like (both fake-button as well)
![image](https://user-images.githubusercontent.com/1755269/65533652-e93aca80-deb2-11e9-8e10-8ba27707ff43.png)
However, there are no hover styles, so we will need those.

Reply:
Oh. got it. let me try.  I'll still need large though